Cargo.lock

```diff
@@ -4,7 +4,7 @@ version = 3
 
 [[package]]
 name = "abq"
-version = "1.7.0"
```

Comment:
@ayazhafiz Does this happen automatically?

Reply:
Yeah. I'm kind of surprised this didn't update before